egs/cnceleb/v1/local/combine_short_segments.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+
+# Copyright 2013  Johns Hopkins University (author: Daniel Povey)
+#           2019  Tsinghua University (author: Jiawen Kang)
+# Apache 2.0
+
+# This script copies and modifies a data directory while combining
+# segments whose duration is lower than a specified minimum segment
+# length.
+#
+# Note: this does not work for the wav.scp, since there is no natural way to
+# concatenate segments; you have to operate on directories that already have
+# features extracted.
+
+# Copied from utils/data/combine_short_segments.sh
```

Comment:
As with the previous script, would you mind explaining what the difference is between this script and the one in utils? 

In general, it's best to try and make things work with existing scripts in the utils/ or steps/ directory, rather than making a new one in local/ if it does almost the same thing. If the utils/ version doesn't quite give you what you want, you might find that you can provide some option to the script that supports the required functionality.

Reply:
Thanks a lot. 
The only difference between this script and the one in utils/ is that we add the '--no-text' options in 'utils/validate_data_dir.sh' because here is no text file. 
It is OK to add an option in 'utils/validate_data_dir.sh' or in 'utils/data/combine_short_segments.sh' to ignore checking text file ? Do you think which one is better ?

Reply:
I think it would be best to add a --no-text option to utils/data/combine_short_segments.sh. Have it default to false, that way the original functionality is maintained. If it is true, then you can add the --no-text option to the call to utils/validate_data_dir.sh.

Reply:
I just had a look at it.  David's suggestion was reasonable, but what I'd like to be done is to add the --no-text option to there combine_short_segments.sh calls validate_data_dir.sh (it will still check the text file if it's there), and when doing:
`utils/apply_map.pl -f 2- $srcdir/text <$dir/utt2utts > $dir/text`
just check that the text file is present in $srcdir before doing that.

Reply:
Thanks a lot. I have updated it.